tool-openssl/ecparam_test.cc

```diff
@@ -200,101 +259,125 @@ TEST_F(EcparamOptionUsageErrorsTest, InvalidOutformTest) {
 // -------------------- OpenSSL Comparison Tests ------------------------------
 
 // Parameterized tests for curve parameter comparison
-class EcparamCurveComparisonTest : public ::testing::Test, public ::testing::WithParamInterface<CurveTestParams> {
-protected:
+class EcparamCurveComparisonTest
+    : public ::testing::Test,
+      public ::testing::WithParamInterface<CurveTestParams> {
+ protected:
   void SetUp() override {
     memset(out_path_tool, '\0', PATH_MAX);
     memset(out_path_openssl, '\0', PATH_MAX);
     tool_executable_path = getenv("AWS_LC_TOOL_EXECUTABLE_PATH");
     openssl_executable_path = getenv("OPENSSL_EXECUTABLE_PATH");
     if (tool_executable_path == nullptr || openssl_executable_path == nullptr) {
-      GTEST_SKIP() << "Skipping test: AWS_LC_TOOL_EXECUTABLE_PATH and/or OPENSSL_EXECUTABLE_PATH environment variables are not set";
+      GTEST_SKIP()
+          << "Skipping test: AWS_LC_TOOL_EXECUTABLE_PATH and/or "
+             "OPENSSL_EXECUTABLE_PATH environment variables are not set";
     }
     ASSERT_GT(createTempFILEpath(out_path_tool), 0u);
     ASSERT_GT(createTempFILEpath(out_path_openssl), 0u);
   }
 
   void TearDown() override {
-    if (strnlen(out_path_tool, PATH_MAX) > 0) {  // Only remove if path was created
+    if (strnlen(out_path_tool, PATH_MAX) >
```

Comment:
Not a priority for this PR, I understand this was only a minor adjustment in the format in this file.

But we should change this to use `OPENSSL_strnlen` for this and similar cases.